imgui_markdown.h

```diff
@@ -230,6 +230,11 @@ ___
 
 namespace ImGui
 {
+    // Configuration - internal use only
```

Comment:
Even if this is internal configuration it should go in the config with the default and marked internal [similarly to how ImGui itself does](https://github.com/ocornut/imgui/blob/master/imgui.h#L1178-L1184). I also don't want to introduce unnamed namespaces under the ImGui namespace as a header.